common/modifiers/hmo_hardcoded.txt

```diff
@@ -109,7 +109,7 @@ nation_rank_0 = {
 	army_weight_modifier = -0.25
 	max_mercenary_stacks = 1
 
-	agressive_expansion_impact = -0.3
+	agressive_expansion_impact = -0.15
```

Comment:
making these a *little* more painful since i was changing stuff in this file

Reply:
i like this change, it will definitely make larger, blobbing players think about expansion sooner than later and might encourage more use of the nation stances and spread out Invention trees that players go down